EnglishFilter/sections/content_blocker.txt

```diff
@@ -85,6 +85,7 @@ cdn.braun634.com^$third-party
 @@||zoover.adtrackers.net/shared/bannerpages/darttagsbanner.aspx
 @@||zoover.bannertrack.net/shared/bannerpages/darttagsbanner.aspx
 !
+@@||satunivers.tv^$generichide
```

Comment:
Could you move it a little lower? `!--- antiblock.org g207---` is a right place.